components/engine_rocks/src/config.rs

```diff
@@ -340,9 +340,9 @@ pub enum BlobRunMode {
 impl From<BlobRunMode> for ConfigValue {
     fn from(mode: BlobRunMode) -> ConfigValue {
         let str_value = match mode {
-            BlobRunMode::Normal => "normal",
-            BlobRunMode::ReadOnly => "read-only",
-            BlobRunMode::Fallback => "fallback",
+            BlobRunMode::Normal => "kNormal",
```

Comment:
the output in the config toml would be `kNormal`?

Reply:
I wonder if we ever export the config or just do import. If we do export, then this may be problematic.

Reply:
It's still normal/read-only/fallback (Verified) @v01dstar @Connor1996 
Because the export of config would use serde serialization/deserialization.